wgsl/index.bs

```diff
@@ -6811,11 +6811,14 @@ That's not a full user-defined function declaration.
     (GLSLstd450Fract)
 
   <tr algorithm="frexp">
-    <td>|T| is [FLOATING]<br>
+    <td>|T| is [FLOATING];<br>
```

Comment:
The semicolons are an attempt to better separate the conditions.  These complex rule sets are visually crammed in a narrow box.